source/sgp_mode/Tasks.h

```diff
@@ -219,8 +259,35 @@ class TaskSet {
       n_succeeds_sym[i]->store(0);
     }
   }
+
+  /**
+   * Input: A boolean identifying a given organism as a host or a symbiont
+   *
+   * Output: The map of square frequencies that matches the organism's type
+   *
+   * Purpose: Returns either hostCalculationTable or symCalculationTable, depending
+   on whether the organism is a host or a symbiont. 
+   */
+std::map<uint32_t, uint32_t> GetSquareFrequencyData(bool Host){
```

Comment:
Let's change the variable name to is_host so that it's both clear that it's a predicate and so it isn't confused with the actual class name